src/components/Superfest/SuperfestPage/StepsBox/StepsBox.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  LeftTextBox,
+  SuperfestPageElementContainer,
+} from '../SuperfestMissionPage.style';
+import { CustomRichBlocks } from 'src/components/Blog';
+import { RootNode } from 'node_modules/@strapi/blocks-react-renderer/dist/BlocksRenderer';
+import { Sequel65Typography } from '../../Superfest.style';
+
+interface StepsBoxProps {
+  steps?: RootNode[];
+  baseUrl: string;
+}
+
+export const StepsBox = ({ steps, baseUrl }: StepsBoxProps) => {
+  return (
+    <SuperfestPageElementContainer>
+      <LeftTextBox>
+        <Sequel65Typography fontSize={'48px'} fontWeight={700}>
+          Steps to complete the mission
+        </Sequel65Typography>
+      </LeftTextBox>
+      <>
+        <CustomRichBlocks
+          id={1}
+          baseUrl={baseUrl}
+          content={steps}
+          variant={'superfest'}
```

Comment:
The string 'superfest' is also being used a lot of times to be passed on or checked in regards of "variant". This could potentially be a const in one single spot (, _as we have done withing menuKeys.ts as an example_)